### PR TITLE
DT-1144: Update Support Destination URLs

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
https://broadworkbench.atlassian.net/browse/DT-1144

## Summary
* Updates the support calls to go through Consent instead of directly to Zendesk
* Adds unit tests for the two components (`RequestForm` and `SupportRequestModal`) that make use of support request functionality.
* Rename the `SupportRequestsPage` component to the file name, `RequestForm`, for consistency
* There are a lot of linting changes, I suggest hiding whitespace changes when reviewing.

Currently blocked on https://github.com/DataBiosphere/consent/pull/2455

###  Testing
To test locally, this needs to be run against a local running consent with the following config setting:
```
services:
  activateSupportNotifications: true
``` 
Visit the Contact Us form on any page, and the Request a New Role from the user profile page. These are the two places where we invoke the new support urls.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/libs/ajax.js [hunk 1]
```diff
@@ -74,11 +74,6 @@ export const fetchAny = async (...args) => {
   return res;
 };
 
-export const getFileNameFromHttpResponse = (response) => {
-  const respHeaders = response.headers;
-  return respHeaders.get('Content-Disposition').split(';')[1].trim().split('=')[1];
-};
-
```
Comment (rushtong) [inline src/libs/ajax.js:81]: Drive-by fix: unused method.

File: src/libs/ajax.js [hunk 2]
```diff
@@ -42,10 +42,6 @@ export const getOntologyUrl = async() => {
   return await Config.getOntologyApiUrl();
 };
 
-export const sleep = (ms) => {
-  return new Promise(resolve => setTimeout(resolve, ms));
-};
-
```
Comment (rushtong) [inline src/libs/ajax.js:48]: Drive-by fix: unused method.

File: src/libs/config.js [hunk 3]
```diff
@@ -17,8 +17,6 @@ export const Config = {
 
   getNihUrl: async () => (await getConfig()).nihUrl,
 
-  getGoogleClientId: async () => (await getConfig()).clientId,
-
```
Comment (rushtong) [inline src/libs/config.js:21]: Drive-by fix: unused method.

**Review by fboulnois — APPROVED** [review]
looks good! 👍

File: src/libs/ajax/Support.js [hunk 4]
```diff
@@ -1,40 +1,36 @@
...
 
-    ticket.request = {
-      requester: { name: name, email: email },
+  createTicket: (name, type, email, subject, description, attachmentToken, url) => {
+    return {
+      name: name,
+      type: type.toUpperCase(),
+      email: email,
       subject: subject,
-      // BEWARE changing the following ids or values! If you change them then you must thoroughly test.
```
Comment (rjohanek) [inline src/libs/ajax/Support.js:13]: what was the purpose of these fields?

**Review by rjohanek — APPROVED** [review]
this looks good!

File: src/libs/ajax/Support.js (hunk 4 above)
Comment (rushtong) [reply src/libs/ajax/Support.js:13]: @raejohanek They are custom field identifiers defined in the Zendesk admin portal that describe the fields displayed in their UI. These were copied over to the API layer here: https://github.com/DataBiosphere/consent/pull/2448/files#diff-310d0997bfbaf3ef54cfda5d6fdd932c172581d9e7e4b38d2a0a44c7f037c91aR48
